VisualRiders.PointOfSale.Project/Services/StaffMembersService.cs

```diff
@@ -0,0 +1,74 @@
+﻿using AutoMapper;
+using VisualRiders.PointOfSale.Project.DTOs;
+using VisualRiders.PointOfSale.Project.Models;
+using VisualRiders.PointOfSale.Project.Repositories;
+
+namespace VisualRiders.PointOfSale.Project.Services;
+
+public class StaffMembersService
+{
+    private readonly StaffMembersRepository _repository;
+    private readonly BusinessEntitiesService _businessEntitiesService;
```

Comment:
Should have used repository instead of service as services mostly communicate with the repository layer

Reply:
Asked a friend for help, he said that lecturer said that 1 service only knows 1 repository.